orchestration/src/test/resources/filteringLooseRequest.json

```diff
@@ -19,7 +19,9 @@
             "role": "user",
             "content": "Hello World! Why is this phrase so famous?"
           }
-        ]
+        ],
+        "defaults" : { },
+        "tools" : [ ]
```

Comment:
(MInor/Question)

This (empty array) looks surprising to me.
I would expect orchestration to complain.
Same goes for empty object.
Is the code tested?

Reply:
Yes, I tested that this is fine. We didn't see an empty field for `defaults` before because the type now changed from `Object` to `Map<String, String>` due to [this spec change](https://github.com/SAP/ai-sdk-java/pull/285/files#diff-e38bdb7287bfc6c2a7f9528fdafb3b46544d6e651d6ca71a5c577bb8ccfd1843R554). The map is initialized as empty map by default, the object used to be simply null, thus this diff.

Reply:
I see `input_params` and `messages_history` are similarly sent empty.